### ISSUE TITLE
Cannot get scene to render properly.

### ISSUE THREAD

**Issue by dirkmittler** [OGRECave/ogitor#49]
Hello.

I'm using a Debian 11 computer with an NVIDIA GeForce RTX 3070 graphics card, and have custom-compiled Ogre 1.12.13, trying to get Ogitor to work with it. The Ogre Sample Browser runs almost all the samples (except for the PN Triangle sample). However, when I load the sample scene within Ogitor, the terrain does not show, the FPS is at zero, and I get flooded with the following message:

```
Application::notify caught an exception: InvalidParametersException: This pass does not have this program type assigned! in getGpuProgramParameters at /home/dirk/Programs/ogre-1.12.13/OgreMain/src/OgrePass.cpp (line 1302)
```

What could I be doing wrong?

Dirk

**Comment by paroj** [comment]
yes, Ogitor's terrain shaders are completely out of sync with Ogre and only support Cg.

However, they should still work on your GPU as long as you stick to the GL RenderSystem (not GL3+).
